app/controllers/api/v1/members_controller.rb

```diff
@@ -0,0 +1,56 @@
+class Api::V1::MembersController < ApplicationController
```

Comment:
This ticket doesn't specify endpoints, so the controller must be an empty file.

```suggestion
module Api
  module V1
    class MembersController < ApplicationController

    end
  end
end
```